src/selfplay/loop.cc

```diff
@@ -123,13 +126,46 @@ void SelfPlayLoop::CmdSetOption(const std::string& name,
 }
 
 void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
+  int winp1   = info.results[0][0] + info.results[0][1];
+  int loosep1 = info.results[2][0] + info.results[2][1];
+  int draws   = info.results[1][0] + info.results[1][1];
+  float perct = -1, elo = 99999;
+  float los   = 99999;
+  if ((winp1 + loosep1 + draws) > 0)
+    perct = (((float)draws) / 2 + winp1) / (winp1 + loosep1 + draws);
```

Comment:
braces around body of conditional operator if it's on a separate line.
use `static_cast<float>()` instead of C-style casts.

Reply:
will do.